### PR TITLE
docs: minor improvements and typo fixes in documentation (DSP-1531)

### PR DESCRIPTION
**Pull request by BalduinLandolt**
resolves DSP-1531

### CODE REVIEW

File: docs/tab_view.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ This class represents a financial grant that was granted to the project.
 
 There can be multiple datasets.
 
-A project hosted on the DSP must have one or multiple datasets.
+A project hosted on the DSP must have at least one datasets.
```
Comment (danielasubotic) [inline docs/tab_view.md:70]: dataset

File: dspMetadataGUI/util/metaDataSet.py [hunk 2]
```diff
@@ -23,9 +23,9 @@
 
 
 class MetaDataSet:
-    """ Representation of a data set.
+    """ Representation of a datset.
```
Comment (danielasubotic) [inline dspMetadataGUI/util/metaDataSet.py:26]: dataset

**Comment by BalduinLandolt** [comment]
möp... now? ^^

**Review by danielasubotic — APPROVED** [review]
möp, yes ^^

**Comment by BalduinLandolt** [comment]
thanks!

**Pull request merged**